src/layers/conv.jl

```diff
@@ -480,6 +480,65 @@ end
 outdims(l::CrossCor, isize) =
   output_size(DenseConvDims(_paddims(isize, size(l.weight)), size(l.weight); stride = l.stride, padding = l.pad, dilation = l.dilation))
 
+"""
+    AdaptiveMaxPool(out)
+Adaptive max pooling layer. `out` is the size of the dimension of the output.
```

Comment:
```suggestion
Adaptive max pooling layer. `out` is the desired output size (batch and channel dimension excluded).
```
Maybe it is better to add a usage example to this docstring

Reply:
ok